accelerator/mqtt_main.c

```diff
@@ -27,6 +27,11 @@ int main(int argc, char *argv[]) {
     return EXIT_FAILURE;
   }
 
+  // Initialize configurations with file value
```

Comment:
What is "file value"?

Reply:
It should be `configuration file` instead of `file value`